services/QuillLMS/spec/factories/admin_infos.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: admin_infos
+#
+#  id                  :bigint           not null, primary key
+#  approval_status     :string
+#  sub_role            :string
+#  verification_reason :text
+#  verification_url    :string
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#  user_id             :bigint           not null
+#
+# Indexes
+#
+#  index_admin_infos_on_user_id  (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (user_id => users.id)
+#
+FactoryBot.define do
+  factory :admin_info do
+    admin { create(:user) }
```

Comment:
While this works, the `create(:user)` adds an unnecessary create call for some build [strategies](https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#build-strategies).

Instead, you can specify the factory [explicitly](https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#specifying-the-factory) and then let factory bot decide whether to create (or just build) the associated object:

```
factory :admin_info do
  association :admin, factory: :user
end
```